lib/situation_room/check/outage.ex

```diff
@@ -0,0 +1,34 @@
+defmodule SituationRoom.Check.Outage do
+  @moduledoc """
+  Module that checks if there is an outage
+  """
+  alias SituationRoom.Outages
+  @behaviour SituationRoom.Check.Behaviour
+
+  @impl true
+  def run(%SituationRoom.Site.Check{} = check) do
+    # This will hold a map of each check that is performed.
+    checklist_of_checks = %{
+      :status_code => SituationRoom.Check.StatusCode.run(check)
+    }
+
+    # Iterate through the map of checks to analyze the results
+    for {k, v} <- checklist_of_checks do
+      case v do
+        {:error} ->
+          do_outage(check, k)
+
+        {:ok} ->
+          nil
+      end
+    end
+
+    {:ok}
+  end
+
+  def do_outage(%SituationRoom.Site.Check{} = check, _reason) do
```

Comment:
Can this be called `create_new_outage()`? And then the method that would mark an outage resolved would be something like `resolve_outage()`?